pytorch/containers/custom_container/README.md

```diff
@@ -0,0 +1,165 @@
+* `BUCKET_ID <YOUR_BUCKET_ID>` - with the bucket id you created above.
+* `IMAGE_REPO_NAME <sonar_pytorch_container>` - where the image will be stored on Cloud Container Registry
+* `IMAGE_TAG <sonar_pytorch>` - a easily identifiable tag for your docker image
+* `IMAGE_URI <gcr.io/$PROJECT_ID/$IMAGE_REPO_NAME:$IMAGE_TAG>` - the complete URI location for Cloud Container Registry
+* `REGION <REGION>` - select a region from [here](https://cloud.google.com/ml-engine/docs/regions) or use the default '`us-central1`'. The region is where the model will be deployed.
+* `JOB_NAME <custom_container_job_...>` - the name of your job running on Cloud ML Engine.
+* `MODEL_NAME <sonar_model>` - the name of your saved model file
+
+```
+# Replace YOUR_PROJECT_ID
+export PROJECT_ID=YOUR_PROJECT_ID
+# Replace BUCKET_ID
+export BUCKET_ID=BUCKET_ID
+export IMAGE_REPO_NAME=sonar_pytorch_container
+export IMAGE_TAG=sonar_pytorch
+export IMAGE_URI=gcr.io/$PROJECT_ID/$IMAGE_REPO_NAME:$IMAGE_TAG
+export REGION=us-central1
+export JOB_NAME=custom_container_job_$(date +%Y%m%d_%H%M%S)
+export MODEL_NAME=sonar_model
+```
+
+# Part 1: Create the model you want to train
+Here we provide an example [model.py](model.py) that trains a PyTorch model
+to predict whether the given sonar signals are bouncing off a metal cylinder or off a cylindrical
+rock.
+
+Open up the [task.py](task.py) to see exactly how the model is called during
+training.
+
+[data_utils.py](data_utils.py) is used to download / load the data
+and exports your trained model and uploads the model to Google Cloud Storage.
+
+The dataset for the model is hosted originally at the UCI Machine Learning Repository. We've 
+[hosted the sonar dataset in Cloud Storage](https://storage.cloud.google.com/cloud-samples-data/ml-engine/sonar/sonar.all-data?organizationId=433637338589&_ga=2.163217084.-1279615720.1534888758)
+for use with this sample.
+
+# Part 2: Create the docker Image
+Open the [Dockerfile](Dockerfile) to see how the Docker image is created that will run on Cloud
+ML Engine.
+
+# Part 3: Build the docker Image
+```
+docker build -f Dockerfile -t $IMAGE_URI ./
+```
+
+# Part 4: Test your docker image locally
+```
+docker run $IMAGE_URI --epochs 1
+```
+
+If it runs successfully, the output should be similar to:
+```
+[1,     6] loss: 0.707
+[1,    12] loss: 0.685
+[1,    18] loss: 0.693
+[1,    24] loss: 0.706
+[1,    30] loss: 0.685
+[1,    36] loss: 0.690
+[1,    42] loss: 0.696
+[1,    48] loss: 0.688
+
+Test set:
+    Average loss: 2.8893
+    Accuracy: 121/208 (58%)
+```
+
+# Part 5: Deploy the docker image to Cloud Container Registry
+You should have configured docker to use Cloud Container Registry, found
+[here](https://cloud.google.com/container-registry/docs/pushing-and-pulling).
+```
+docker push $IMAGE_URI
+```
+
+# Part 6: Submit your training job
+Creates a config file for your job request.
+```
+cat > config.yaml <<EOF
```

Comment:
Why not include the sample `config.yaml` as one of the files in the repo?

Reply:
I didn't want to have to tell them to open the config and change the `$IMAGE_URI`, since it can be rather long.